src/main/java/com/codeborne/selenide/SelenideElement.java

```diff
@@ -1109,9 +1115,23 @@ public interface SelenideElement extends WebElement, WrapsDriver, WrapsElement,
   @CanIgnoreReturnValue
   SelenideElement scrollIntoView(String scrollIntoViewOptions);
```

Comment:
If we introduce typed method let's deprecate or even remove this one (which accept `String`)

Reply:
done.